demo/apisix/conf/apisix.yaml

```diff
@@ -264,6 +264,8 @@ plugin_configs:
           failures: 3  # should be var: JMAP_CIRCUIT_BREAKER_MAXERRORS
         healthy:
           successes: 1
+      client-control:
+        max_body_size: 20000000
```

Comment:
Should be lower e.g. 5000000? cf https://github.com/linagora/james-project/issues/5074#issuecomment-1956870172

Reply:
1MB for everything.
20MB default for uploads...